web/src/utils/getFormattedBalance.ts

```diff
@@ -1,7 +1,7 @@
 import { formatETH, formatPNK } from "./format";
 
-export const getFormattedBalance = (balanceData: any, token: any) => {
-  if (!balanceData) return undefined;
-  if (token?.symbol === "PNK") return formatPNK(balanceData.value);
-  return formatETH(balanceData.value);
+export const getFormattedBalance = (balanceData: bigint | undefined, token: any) => {
+  if (balanceData === undefined) return undefined;
```

Comment:
we can use the `isUndefined()` util function from the "utils/index" folder instead